packages/app/package.json

```diff
@@ -30,6 +30,7 @@
     "@hookform/resolvers": "^2.9.8",
     "@xstate/react": "^3.0.1",
     "buffer": "^6.0.3",
+    "cross-fetch": "^3.1.5",
```

Comment:
Why do we need cross-fetch?

Reply:
I got the idea from `swayswap`, makes fetch platform agnostic allowing to use in node envs, like testing in jest